edm4hep.yaml

```
@@ -780,6 +780,36 @@ datatypes:
     OneToOneRelations:
       - edm4hep::Track track           // the corresponding track
 
+
+  #===== Generator related data =====
+
+  #---------- GeneratorEventParameters
+  edm4hep::GeneratorEventParameters:
+    Description: "Generator event parameters"
+    Author: "EDM4hep authors"
+    Members:
+      - double event_scale              // event scale
+      - double alphaQED                 // alpha_QED
+      - double alphaQCD                 // alpha_QCD
+      - int signal_process_id           // id of signal process
+      - double sqrt_s  [GeV]            // sqrt(s)
+    VectorMembers:
+      - double crossSections [pb]       // list of cross sections
+      - double crossSectionErrors       // list of cross section errors
+    OneToManyRelations:
+      - edm4hep::MCParticle signalVertex // pointing into MCParticle collection
+
+  #---------- GeneratorPdfInfo
+  edm4hep::GeneratorPdfInfo:
+    Description: "Generator pdf information"
+    Author: "EDM4hep authors"
+    Members:
+      - std::array<int, 2> parton_id      // Parton PDG id
+      - std::array<int, 2> id             // LHAPDF PDF id (see https://lhapdf.hepforge.org/pdfsets.html)
```

```suggestion
      - std::array<int, 2> lhapdfId             // LHAPDF PDF id (see https://lhapdf.hepforge.org/pdfsets.html)
```
(Or similar), to make it inferable from the name what you will get? Also if we keep it at `id` we will have an `id()` on the handle that will give you the `podio::ObjectId`, whereas the `getId()` method will get you this array.